src/components/accounts/CreateAccount.js

```diff
@@ -133,18 +167,59 @@ class CreateAccount extends Component {
             loader,
             accountId,
             invalidNearDrop,
-            showTerms,
-            termsChecked,
-            privacyChecked,
+            termsAccepted,
+            whereToBuy
         } = this.state
 
-        const { localAlert, mainLoader, checkNewAccount, resetAccount, clearLocalAlert } = this.props
+        const { localAlert, mainLoader, checkNewAccount, resetAccount, clearLocalAlert, fundingContract, fundingKey } = this.props;
+        const hasFunding = fundingContract && fundingKey;
```

Comment:
Does this represent e.g. `isLinkDrop`? Or could hasFunding be true for reasons other than a linkdrop?